geoips/commandline/geoips_get.py

```diff
@@ -0,0 +1,342 @@
+# # # This source code is protected under the license referenced at
+# # # https://github.com/NRLMMD-GEOIPS.
+
+"""GeoIPS CLI "get" command.
+
+Retrieves the appropriate family/interface/package/plugin based on the args provided.
+"""
+
+from importlib import metadata, resources, import_module
+
+import yaml
+
+from geoips.commandline.geoips_command import GeoipsCommand, GeoipsExecutableCommand
+from geoips.create_plugin_registries import format_docstring
+from geoips import interfaces
+
+
+class GeoipsGetFamily(GeoipsExecutableCommand):
```

Comment:
Should this be removed?  Is this geoips_describe now?

Reply:
I'm not sure. @evrose54?

Reply:
This is out of sync. Main has GeoIPS describe. Unless there were any new CLI updates included in this PR, which to my recollection there weren't, I'd remove this.

Reply:
Just remove the entire geoips_get.py file, right?

Reply:
I am removing it!!